src/adagrams.js

```diff
@@ -1,8 +1,68 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const alphabet = 'AAAAAAAAABBCCDDDDEEEEEEEEEEEEFFGGGHHIJKLLLLMMNNNNNNOOOOOOOOPPQRRRRRRSSSSTTTTTTUUUUVVWWXYYZ';
+    const lettersInHand = [];
+
+    for (let i = 0; i < 10; i++) {
```

Comment:
There's a small bug here. Consider how you could make sure you don't pick the same letter 10 times (however unlikely it is).